requirements.txt

```diff
@@ -1,2 +1,2 @@
-pulpcore>=3.49.0,<3.70
+pulpcore>=3.70.0,<3.85
```

Comment:
Can we remain compatibility back to 3.49? I'm not sure.

Reply:
I've reverted it. Thanks for the suggestion.

Reply:
Based on matrix convo I think we can leave the lower bound the same (and we should). See how Python did it here: https://github.com/pulp/pulp_python/pull/791/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R30

Reply:
Oh you already handled it! Nice! :100: